docs/rules/expect.md

```diff
@@ -0,0 +1,141 @@
+# expect-type/expect
+
+💼 This rule is enabled in the ✅ `recommended` config.
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+💭 This rule requires type information.
+
+<!-- end auto-generated rule header -->
+
+Enforces that types indicated in special comments match the types of code values.
+
+## Comment Types
+
+The following kinds of comments are supported:
+
+<!-- Markdownlint doesn't seem to understand the heading IDs... -->
+<!-- markdownlint-disable link-fragments -->
+
+- [`^?`](#twoslash)
```

Comment:
Suggest something like:

    - [`^?` (Twoslash Syntax)](#twoslash)

Since this one isn't really pronounceable.